src/main/java/io/redlink/more/data/schedule/SchedulerUtils.java

```diff
@@ -98,28 +103,27 @@ public static Instant shiftStartIfObservationAlreadyEnded(Instant start, List<Ob
                 .filter(scheduleEvent -> scheduleEvent.getType().equals(RelativeEvent.TYPE))
                 .map(r -> ((RelativeEvent) r).getDtend())
                 .filter(relativeDate -> relativeDate.getOffset().getValue() == 1)
-                .map(relativeDate -> start.atZone(relativeDate.getZoneId()).withHour(relativeDate.getHours()).withMinute(relativeDate.getMinutes()).withSecond(0).withNano(0).toInstant())
```

Comment:
In the original, we're using `relativeDate.getZoneId()`.

@drtyyj - could you check what implications this change has?